apps/pay/config/config.ts

```diff
@@ -36,9 +35,11 @@ export const getClientSideGqlConfig = () => {
 }
 
 export const getClientSidePayDomain = () => {
-  if (env.NEXT_PUBLIC_PAY_DOMAIN || typeof window === "undefined") {
+  if (env.NEXT_PUBLIC_PAY_DOMAIN) {
     return env.NEXT_PUBLIC_PAY_DOMAIN
+  } else if (typeof window !== "undefined") {
+    return new URL(window.location.href).hostname.split(".").slice(-2).join(".")
```

Comment:
```suggestion
    return new URL(window.location.href).hostname.split(".").slice(-2).join(".") // Return the last two parts of the hostname (e.g. "blink.sv")
```

feels like this comment is useful becuase it's not clear otherwise what this function aims for